_inc/client/writing/custom-content-types.jsx

```diff
@@ -124,32 +134,44 @@ export class CustomContentTypes extends React.Component {
 						disabled={ this.props.isSavingAnyOption( 'jetpack_portfolio' ) || disabledByOverride }
 						onChange={ this.handlePortfolioToggleChange }
 						disabledReason={ disabledReason }
-						>
-						<span className="jp-form-toggle-explanation">
-							{
-								__( 'Portfolios' )
-							}
-						</span>
+					>
+						<span className="jp-form-toggle-explanation">{ __( 'Portfolios' ) }</span>
 					</CompactFormToggle>
 					<FormFieldset>
 						<p className="jp-form-setting-explanation">
-							{
-								__( 'Portfolios shortcode: [portfolio]' )
-							}
+							{ __( 'Portfolios shortcode: [portfolio]' ) }
 						</p>
 					</FormFieldset>
 				</SettingsGroup>
+				{ this.props.portfolioActive && (
+					<CompactCard
+						className="jp-settings-card__configure-link"
+						href={ `${ this.props.siteAdminUrl }post-new.php?post_type=jetpack-portfolio` }
+					>
+						{ __( 'Add a portfolio' ) }
```

Comment:
Maybe it should say "Add a portfolio item" here, since we are not adding a full portfolio?